### ISSUE TITLE
twisted.conch.ssh.client.knownhosts should be ported to Python3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8697]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8697|
|Type|enhancement|
|Created|2016-07-27 13:01:57Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8697 8697
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469624517594368 1469624517594368
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_1"></a>|@einarfd commented|
|-|-|

A merge request fixing this can be found at https://github.com/twisted/twisted/pull/431

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset a7dcf6639157c2d68436c57bbadb3827280670be
```
#!CommitTicketReference repository="" revision="a7dcf6639157c2d68436c57bbadb3827280670be"
Merge pull request #433 from twisted/8697-einarfd-py3-conch-client-knownhosts

Author: einarfd
Reviewer: rodrigc
Fixes: #8697

Port twisted.conch.client.knownhosts to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
